pychunkedgraph/io/edges.py

```diff
@@ -36,38 +36,37 @@ def deserialize(edges_message: EdgesMsg) -> Tuple[np.ndarray, np.ndarray, np.nda
     return Edges(sv_ids1, sv_ids2, affinities=affinities, areas=areas)
 
 
-def _decompress_edges(content: bytes) -> Dict:
+def _parse_edges(compressed: List[bytes]) -> List[Dict]:
     zdc = zstd.ZstdDecompressor()
-    chunk_edges = ChunkEdgesMsg()
-    chunk_edges.ParseFromString(zdc.decompressobj().decompress(content))
-
-    # in, between and cross
-    edges_dict = {}
-    edges_dict[EDGE_TYPES.in_chunk] = deserialize(chunk_edges.in_chunk)
-    edges_dict[EDGE_TYPES.between_chunk] = deserialize(chunk_edges.between_chunk)
-    edges_dict[EDGE_TYPES.cross_chunk] = deserialize(chunk_edges.cross_chunk)
-    return edges_dict
+    decompressed = zdc.multi_decompress_to_buffer(compressed, threads=-1)
```

Comment:
We discussed today and @sdorkenw want to have the number of threads configurable, @akhileshh, maybe read the number of thread from a env so we can control it at deployment time?